staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -123,81 +120,69 @@ func newNoxuValidationCRDs(scope apiextensionsv1beta1.ResourceScope) []*apiexten
 					},
 				},
 			},
-			"delta": {
-				Description: "Delta is a string with a maximum length of 5 or a number with a minimum value of 0",
-				AnyOf: []apiextensionsv1beta1.JSONSchemaProps{
-					{
-						Type:      "string",
-						MaxLength: int64Ptr(5),
-					},
-					{
-						Type:    "number",
-						Minimum: float64Ptr(0),
-					},
-				},
-			},
```

Comment:
Impossible to follow these removals.

Reply:
I remember you asked me on slack. Yes, this is not structural. If you have removed the delta test code as well, I think we are fine.